driver.c

```diff
@@ -142,8 +133,7 @@ int main(int argc, char **argv)
         break;
     }
     default:
-        printf("Abnormal: unknown request: %d\n", req);
-        return -1;
+        FATAL(-1, "Abnormal: unknown request: %d\n", req);
         break;
```

Comment:
Since we introduce macro `FATAL`, it is not necessary to append `break` statement right after that.